src/Employer/Employer.Web/Orchestrators/Part1/LocationOrchestrator.cs

```diff
@@ -152,41 +179,86 @@ private async Task<LocationViewModel> GetViewModelAsync(Vacancy vacancy, bool? h
             var vm = new LocationViewModel();
             vm.PageInfo = Utility.GetPartOnePageInfo(vacancy);
 
-            vm.LegalEntityLocation = legalEntity.Address.ToString();
-
+            vm.LocationName = legalEntity.Address.ToString();
+            vm.OtherLocationsAddress = employerProfile?.OtherLocations;
```

Comment:
If OtherLocations is null then set an empty list instead.  Then we don't have to do null checks.